kisanola/tech-community-frontend
# Add error monitoring and reporting

#### What does this PR do?
This PR adds Sentry error monitoring and reporting

#### Description of Task to be completed?
~ Install the package
~ Initialize Sentry from the application entry point

#### How should this be manually tested?
Fetch and checkout to `ch-add-error-monitoring`
Call an undefined function on any element click event, start the server and click on the button to produce the error.
Then navigate to `https://sentry.io/organizations/tech-communities/issues/?project=1820599` to see the actual report

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/46924450/68876115-9c35e080-070c-11ea-9bba-03e5009ce95f.png)

#### Questions:
N/A
